### PR TITLE
Post-merge content updates

### PR DESCRIPTION
**Pull request by jmcook1186**
## Description

Merges all content changes required to update ethereum.org on the day of The Merge.

This will be the long-running feature branch to address #7075. All related PRs should merge into this base branch.

Preview deploy:
https://build-0453fa18-ad81-4a80-9860-abd61958d0df.gtsb.io/en/upgrades/merge

## Related Issue

#7075

### CODE REVIEW

File: src/content/developers/tutorials/hello-world-smart-contract-fullstack/index.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ skill: beginner
 lang: en
 sidebar: true
 published: 2021-10-25
+postMergeBannerTranslation: page-upgrades-post-merge-banner-tutorial-ood
```
Comment (samajammin) [inline src/content/developers/tutorials/hello-world-smart-contract-fullstack/index.md:23]: Should a banner be displaying for these tutorial pages? I'm not seeing anything show up, e.g.
https://build-eae1f9b7-2544-4fb3-9d3f-abe2b5e5c4e7.gtsb.io/en/developers/tutorials/hello-world-smart-contract-fullstack/

File: src/content/upgrades/beacon-chain/index.md [hunk 2]
```diff
@@ -7,51 +7,43 @@ sidebar: true
 image: ../../../assets/upgrades/core.png
 summaryPoint1: The Beacon Chain doesn't change anything about the Ethereum we use today.
```
Comment (samajammin) [inline src/content/upgrades/beacon-chain/index.md:8]: > The Beacon Chain doesn't change anything about the Ethereum we use today.

We should change or remove this.

File: src/content/upgrades/beacon-chain/index.md [hunk 3]
```diff
@@ -7,51 +7,43 @@ sidebar: true
...
 summaryPoint2: It introduced proof-of-stake to the Ethereum ecosystem.
-summaryPoint3: It will coordinate the network, serving as the consensus layer.
-summaryPoint4: It is an essential precursor to upcoming scaling upgrades, such as sharding.
+summaryPoint3: It was merged with the original Ethereum proof-of-work chain in 2022.
+summaryPoint4: The consensus logic and block gossip protocol introduced by the Beacon Chain now secures Ethereum.
```
Comment (samajammin) [inline src/content/upgrades/beacon-chain/index.md:11]:
```suggestion
summaryPoint4: The Beacon Chain introduced consensus logic and block gossip protocol which now secures Ethereum.
```
Not sure if this is great but I feel like re-wording this to more of an active voice would be better.

File: src/content/upgrades/merge/index.md [hunk 4]
```diff
@@ -1,113 +1,98 @@
...
 lang: en
 template: upgrade
 sidebar: true
 image: ../../../assets/upgrades/merge.png
-summaryPoint1: Soon, the current Ethereum Mainnet will merge with the Beacon Chain proof-of-stake system.
-summaryPoint2: This will mark the end of proof-of-work for Ethereum, and the full transition to proof-of-stake.
-summaryPoint3: This sets the stage for future scaling upgrades including sharding.
-summaryPoint4: The Merge will reduce Ethereum's energy consumption by ~99.95%.
+summaryPoint1: Ethereum Mainnet uses proof-of-stake, but this wasn't always the case.
+summaryPoint2: The transition from the original proof-of-work mechanism to proof-of-stake was called The Merge.
```
Comment (samajammin) [inline src/content/upgrades/merge/index.md:9]:
```suggestion
summaryPoint2: The upgrade from the original proof-of-work mechanism to proof-of-stake was called The Merge.
```
Not a strong preference here but "upgrade" might be more explicit on how the transition happened (or else explicitly mention a hard fork?).

File: src/content/upgrades/merge/index.md [hunk 5]
```diff
@@ -1,113 +1,98 @@
...
+summaryPoint3: The Merge refers to the original Ethereum Mainnet becoming part of a separate proof-of-stake blockchain called the Beacon Chain, now existing as one chain.
```
Comment (samajammin) [inline src/content/upgrades/merge/index.md:10]:
```suggestion
summaryPoint3: The Merge refers to the original Ethereum Mainnet merging with a separate proof-of-stake blockchain called the Beacon Chain, now existing as one chain.
```
Drive the wording home? 😄

File: src/content/developers/tutorials/hello-world-smart-contract-fullstack/index.md (hunk 1 above)
Comment (corwintines) [reply src/content/developers/tutorials/hello-world-smart-contract-fullstack/index.md:23]: Fixed this and pushed into this branch. Should be good to test now

**Pull request merged**